src/corehost/cli/runtime_config.cpp

```diff
@@ -201,14 +280,60 @@ bool runtime_config_t::ensure_parsed()
         trace::verbose(_X("UTF-8 BOM skipped while reading [%s]"), m_path.c_str());
     }
 
+    bool rc = true;
     try
     {
         const auto root = json_value::parse(file);
         const auto& json = root.as_object();
         const auto iter = json.find(_X("runtimeOptions"));
         if (iter != json.end())
         {
-            parse_opts(iter->second);
+            rc = parse_opts(iter->second);
+
+            if (rc)
+            {
+                if (app_config == nullptr)
```

Comment:
This check and proceeding comment are confusing.  if `app_config == nullptr` and `// For the app, populate the additionalFrameworks`.

Is this supposed to be `if (app_config != nullptr)`?

Reply:
It is correct, but yes perhaps confusing. If app_config is nullptr, then there is app_config yet, so we must be the app.